shaky/src/main/java/com/linkedin/android/shaky/ShakeDelegate.java

```diff
@@ -40,19 +40,32 @@
     public static final int SENSITIVITY_MEDIUM = 23;
     public static final int SENSITIVITY_HARD = 24;
 
+    @SensitivityLevel
+    private static int sensitivityLevel = SENSITIVITY_MEDIUM;
     /**
      * @return true if shake detection should be enabled, false otherwise
      */
     public boolean isEnabled() {
         return true;
     }
 
+    /**
+     * @return true if setting is accessible in shaky UI, false otherwise
+     */
+    public boolean isSettingEnabled() {
+        return false;
+    }
+
     /**
      * @return desired sensitivity level, defaults to 'medium'
      */
     @SensitivityLevel
     public int getSensitivityLevel() {
-        return SENSITIVITY_MEDIUM;
+        return sensitivityLevel;
+    }
+
+    public void setSensitivityLevel(@SensitivityLevel int sensitivityLevel) {
```

Comment:
Lets add some comments on this describing how people should override it to save the value locally, as changes do not persisty by default. Seems like a pretty big gotcha so lets try to make it clear